### PR TITLE
fix : Support variable Usages Separated by Commas without Whitespaces…

### PR DESCRIPTION
**Pull request by ap891843**
fix : Support variable Usages Separated by Commas without Whitespaces… #665

Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4 [hunk 1]
```diff
@@ -793,7 +793,7 @@ callUsingPhrase
    ;
 
 callUsingParameter
-   : callByReferencePhrase | callByValuePhrase | callByContentPhrase
+   : (callByReferencePhrase | callByValuePhrase | callByContentPhrase) COMMACHAR?
```
Comment (grianbrcom) [inline server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4:796]: The change will fix the error, but this grammar matches a lot of cases:
```
VAR1, VAR2
VAR1, VAR2,
VAR1 VAR2
```
Probably, it is not correct.

**Review by grianbrcom — COMMENTED** [review]
I need COBOL guru advice.

File: server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4 (hunk 1 above)
Comment (temanbrcom) [reply server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4:796]: I would expect it to be:
```
callUsingPhrase
   : USING callUsingParameter (COMMACHAR? callUsingParameter)*
   ;
```
Should be checked that the positive tests pass and the loose comma at the end produces a syntax error

**Review by temanbrcom — APPROVED** [review]
🌊

**Review by grianbrcom — APPROVED** [review]
:cactus:

**Pull request merged**